cypress/support/pages/home.pageObject.js

```diff
@@ -3,9 +3,20 @@ import PageObject from '../PageObject';
 class homePageObject extends PageObject {
   url = '/#/';
 
-  get usernameLink() {
-    return cy.getByDataCy('/profile/');
+  usernameLink(username) {
```

Comment:
The `usernameLink` method can be confusing as it checks if the link contains the username, but it doesn't return anything. Consider renaming it to `checkUsernameLink` or similar to better reflect its functionality.